src/ResourceManager/Resources/Commands.Resources/Models.Authorization/AuthorizationClient.cs

```diff
@@ -59,13 +60,7 @@ static AuthorizationClient()
         public AuthorizationClient(IAzureContext context)
         {
             ActiveDirectoryClient = new ActiveDirectoryClient(context);
-            AuthorizationManagementClient = AzureSession.Instance.ClientFactory
-#if !NETSTANDARD
-                .CreateClient<AuthorizationManagementClient>
-#else
-                .CreateArmClient<AuthorizationManagementClient>
-#endif
-                (context, AzureEnvironment.Endpoint.ResourceManager);
+            AuthorizationManagementClient = AzureSession.Instance.ClientFactory.CreateArmClient<AuthorizationManagementClient>(context, AzureEnvironment.Endpoint.ResourceManager);
         }
```

Comment:
Because with the new 'Microsoft.Azure.Management.Authorization' package form azure-sdk-for-net, We need to use CreateArmClient for '!NETSTANDARD' situation and then we can pass the test case. So I deleted the 'if else' selection.